models/articles.models.js

```diff
@@ -14,12 +23,17 @@ exports.fetchArticles = () => {
   FROM articles
   JOIN comments
   ON articles.article_id = comments.article_id
-  GROUP BY articles.article_id
-  ORDER BY articles.created_at DESC`
-    )
-    .then(({ rows }) => {
-      return rows;
-    });
+  GROUP BY articles.article_id`;
+
+  if (!validSortBy.includes(sort_by) || !validOrder.includes(order)) {
+    return Promise.reject({ status: 400, msg: 'Bad Request' });
```

Comment:
If you wanted to make your error messages more specific you could deal with each of these conditions separately and return a different msg for both depending on what the issue was. Not a necessity, but maybe nice to have!